<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>601</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{	padding:0px;
	margin:0px;
	background-color:white;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">margin: 0px;
padding:0px;
background:rgb(213, 174, 255);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>831</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="root_layout">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="title_bar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>831</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(213, 174, 255)

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="minimize_btn">
         <property name="geometry">
          <rect>
           <x>670</x>
           <y>0</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#minimize_btn{
						border-image:url('C:/Users/THING007/Documents/playerUI/min.png');

						}
#minimize_btn:hover{
						border-image:url('C:/Users/THING007/Documents/playerUI/min2.png');

						}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="maximize_btn">
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>0</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#maximize_btn{
						border-image:url('C:/Users/THING007/Documents/playerUI/max.png');
}
#maximize_btn:hover{
						border-image:url('C:/Users/THING007/Documents/playerUI/max2.png');
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="close_btn">
         <property name="geometry">
          <rect>
           <x>770</x>
           <y>0</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#close_btn{
						border-image:url('C:/Users/THING007/Documents/playerUI/cross.png');
}
#close_btn:hover{
						border-image:url('C:/Users/THING007/Documents/playerUI/cross2.png');
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="logo">
         <property name="geometry">
          <rect>
           <x>-6</x>
           <y>-3</y>
           <width>341</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>360</y>
          <width>821</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QSlider" name="seek_slider">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>1</y>
           <width>791</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
    
    height: 3px;
    background:black ;
    margin: 1.5px 0;
}

QSlider::handle:horizontal {
    background:silver;
    border:1px solid black;
    width: 18px;
    margin: -7px 0;
    border-radius: 9.4px;
    margin-top: -9px;
}

QSlider::add-page:horizontal{
    border-radius: 3.4px;
    
    background: rgb(138, 192, 255);
}

QSlider::sub-page:horizontal {
    border-radius: 5.4px;
    margin-top:-2px;
    margin-bottom:-2px;
    background: rgb(213, 174, 255);
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="play_btn">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>30</y>
           <width>80</width>
           <height>80</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#play_btn{
								border-image:url('C:/Users/THING007/Documents/playerUI/play1.png');
}
#play_btn:hover{
								border-image:url('C:/Users/THING007/Documents/playerUI/play2.png');
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>89</width>
           <height>89</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="rewind_btn">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>46</y>
           <width>53</width>
           <height>51</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>43</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>57</width>
           <height>51</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#rewind_btn{
					border-image:url('C:/Users/THING007/Documents/playerUI/pre.png');
					}
#rewind_btn:hover{
					border-image:url('C:/Users/THING007/Documents/playerUI/pre2.png');
					}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>53</width>
           <height>53</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QSlider" name="vol_slider">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>60</y>
           <width>160</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
    
    height: 3px;
    background:black ;
    margin: 1.5px 0;
}

QSlider::handle:horizontal {
    background:silver;
    border:1px solid black;
    width: 18px;
    margin: -7px 0;
    border-radius: 9.4px;
    margin-top: -9px;
}

QSlider::add-page:horizontal{
    border-radius: 3.4px;
    
    background:  rgb(138, 192, 255);
}

QSlider::sub-page:horizontal {
    border-radius: 5.4px;
    margin-top:-2px;
    margin-bottom:-2px;
    background: rgb(213, 174, 255);
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="camera_btn">
         <property name="geometry">
          <rect>
           <x>759</x>
           <y>70</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#camera_btn{
						border-image:url('C:/Users/THING007/Documents/playerUI/camera.png');
					  }
#camera_btn:hover{
						border-image:url('C:/Users/THING007/Documents/playerUI/camera2.png');
					  }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="forward_btn">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>45</y>
           <width>53</width>
           <height>51</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>43</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>57</width>
           <height>51</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#forward_btn{
						border-image:url('C:/Users/THING007/Documents/playerUI/next.png');
						}
#forward_btn:hover{
						border-image:url('C:/Users/THING007/Documents/playerUI/next2.png');
						}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>53</width>
           <height>53</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="add_btn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#add_btn{
						border-image:url('C:/Users/THING007/Documents/playerUI/add.png');
					  }
#add_btn:hover{
						border-image:url('C:/Users/THING007/Documents/playerUI/add2.png');
					  }</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="song_elapsed_time">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tekton Pro</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
        <widget class="QLabel" name="song_duration">
         <property name="geometry">
          <rect>
           <x>777</x>
           <y>30</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Tekton Pro</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pause_btn">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>30</y>
           <width>80</width>
           <height>80</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">#pause_btn{
								border-image:url('C:/Users/THING007/Documents/playerUI/pause1.png');
}
#pause_btn:hover{
								border-image:url('C:/Users/THING007/Documents/playerUI/pause2.png');
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>89</width>
           <height>89</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="song_name">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>100</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tekton Pro</family>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string> No Song selected</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
       <widget class="QLabel" name="author_name">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>160</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tekton Pro</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <strikeout>false</strikeout>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>NA</string>
        </property>
       </widget>
       <widget class="QLabel" name="thumbnail">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>190</width>
          <height>190</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 5px solid  rgba(0,0,0,0.5);
border-radius:5px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="pixmap">
         <pixmap>../../../../Visionplayer/Icons/thumb.gif</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="year">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>200</y>
          <width>461</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tekton Pro</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <strikeout>false</strikeout>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>NA</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="acrcloud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Powered by ACRCloud and OpenCV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_acr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#add_acr{
				
				border-image:url('C:/Users/THING007/Documents/playerUI/acr.png');
			   }
#add_acr:hover{
				
				border-image:url('C:/Users/THING007/Documents/playerUI/acr2.png');
			   }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="acr1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>20</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="acr2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tekton Pro</family>
      <pointsize>20</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
